spec/poros/food_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "Food" do
+  describe 'attributes' do
+    xit 'can be created from a hash' do
+
+    end
+    xit 'has ' do
+
+    end
+  end
+end
```

Comment:
You started building out this food poro spec. In addition to completing this spec, we'd also like to see you create tests for all other classes you created -- facade, service.